spring-amqp/src/main/java/org/springframework/amqp/core/AcknowledgeMode.java

```diff
@@ -17,21 +17,53 @@
 package org.springframework.amqp.core;
 
 /**
+ * Acknowledgment modes supported by the listener container.
+ *
  * @author Dave Syer
  *
  */
 public enum AcknowledgeMode {
 
-	NONE, MANUAL, AUTO;
+	/**
+	 * No acks - {@code autoAck=true} in {@code Channe.basicConsume()}.
```

Comment:
Typo: Channe**l**
Will fix on merge together with the `@author` effort respect :smile: